packages/orchestration/src/exos/chain-hub.js

```diff
@@ -40,11 +41,24 @@ export const DenomDetailShape = M.splitRecord(
   { brand: BrandShape },
 );
 
-/** agoricNames key for ChainInfo hub */
+// TODO refactor into an enum-ish object
+/**
+ * agoricNames key for ChainInfo hub
+ *
+ * @internal
```

Comment:
These agoricNames keys don't seem internal to me. They show up all the way to clients via vstorage.

Reply:
good point, I'll revert those. I still think they should go into an enum, but that can wait